.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ mainBuildFilters: &mainBuildFilters
     branches:
       only:
         - develop
-        - release/12.0.0
+        - /^release\/\d*\.\d*\.\d*$/
```

Comment:
```suggestion
        - /^release\/\d+\.\d+\.\d+$/
```

Should be at least one number in each position, right?

Reply:
ooh, yes there should be!

Reply:
Updated